js/apps/admin-ui/src/context/RealmsContext.tsx

```diff
@@ -5,25 +5,31 @@ import { createNamedContext, useRequiredContext } from "ui-shared";
 import { keycloak } from "../keycloak";
 import { useFetch } from "../utils/useFetch";
 import { fetchAdminUI } from "./auth/admin-ui-endpoint";
+import { sortBy } from "lodash-es";
 
 type RealmsContextProps = {
   /** A list of all the realms. */
-  realms: string[];
+  realms: RealmNameRepresentation[];
   /** Refreshes the realms with the latest information. */
   refresh: () => Promise<void>;
 };
 
+export interface RealmNameRepresentation {
+  name: string;
+  displayName: string | undefined;
```

Comment:
This can be made optional, rather than overloading it with `undefined`.
```suggestion
  displayName?: string;
```